exportimportconfig-sample.py

```diff
@@ -34,6 +34,8 @@
         'userb': 'user2',
 }
 
+DEFAULT_ASSIGNEE = 'user3'
+
```

Comment:
@mrts It completely makes sense to have default values for priority, issue type, assignee and reported, but isn't it better to add them all in `exportimportconfig-sample.py` file like `DEFAULT_ASSIGNEE`